<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>763</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1230</width>
    <height>763</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1230</width>
    <height>763</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>resources/icon1.png</normaloff>resources/icon1.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="imgLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1080</width>
      <height>720</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(199, 199, 199);
border:1px solid rgb(144, 144, 144);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="roiLabel1">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>10</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgb(199, 199, 199);
	border:1px solid rgb(144, 144, 144);
}
QLabel:hover{
	background-color: rgb(229, 241, 251);
	border:1px solid rgb(0, 120, 215);
}
QLabel:pressed{
	background-color: rgb(229, 241, 251);
	border:1px solid rgb(100, 120, 215);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>resources/icon1.ico</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="roiLabel2">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>140</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgb(199, 199, 199);
	border:1px solid rgb(144, 144, 144);
}
QLabel:hover{
	background-color: rgb(229, 241, 251);
	border:1px solid rgb(0, 120, 215);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="roiLabel3">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>270</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgb(199, 199, 199);
	border:1px solid rgb(144, 144, 144);
}
QLabel:hover{
	background-color: rgb(229, 241, 251);
	border:1px solid rgb(0, 120, 215);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="roiLabel4">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>400</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgb(199, 199, 199);
	border:1px solid rgb(144, 144, 144);
}
QLabel:hover{
	background-color: rgb(229, 241, 251);
	border:1px solid rgb(0, 120, 215);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="name1">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>110</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>用户0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="name2">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>240</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>用户1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="name3">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>370</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>用户2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="name4">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>500</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>用户3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="button2">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>680</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button1">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>640</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>保存人脸</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line1">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>530</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
   </widget>
   <widget class="QLineEdit" name="line2">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>560</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1230</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu1">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action1"/>
   </widget>
   <widget class="QMenu" name="menu2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="action2"/>
   </widget>
   <addaction name="menu1"/>
   <addaction name="menu2"/>
  </widget>
  <action name="action1">
   <property name="text">
    <string>所有用户</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
